cmd/juju/application/deploy.go

```diff
@@ -662,9 +663,6 @@ func (c *DeployCommand) SetFlags(f *gnuflag.FlagSet) {
 }
 
 func (c *DeployCommand) Init(args []string) error {
-	if c.Force && c.Series == "" && c.PlacementSpec == "" {
```

Comment:
Do we know why PlacementSpec is in this list?

Reply:
Not a clue - it's not tested :disappointed: